app/assets/stylesheets/objects/o-article.scss

```diff
@@ -0,0 +1,63 @@
+.o-article {
+  &__container {
+    min-width: 300px;
+    padding: 0 5%;
+    margin: 0 auto 35px;
+  }
+}
+
+@media only screen and (min-width: 500px) {
+  .o-article {
+    &__container {
+      width: 480px;
+    }
+  }
+}
+
+@media screen and (min-width: 768px) {
+  .o-article {
+    &__container {
+      width: 668px;
+    }
+  }
+}
+
+@media screen and (min-width: 1024px) {
+  .o-article {
+    &__container {
+      width: 750px;
+      padding: 0 30px;
+    }
+  }
+}
+
+// - Naming files so that they do not contain BEM nomenclature 
```

Comment:
I would probably move these comments into your personal notes if you think it helps, then they can be removed from this file